packages/dependency-extraction-webpack-plugin/src/index.js

```diff
@@ -1,4 +1,5 @@
 const WPDependencyExtractionWebpackPlugin = require( '@wordpress/dependency-extraction-webpack-plugin' );
+const packages = require( '../assets/packages' );
```

Comment:
I may have set things up wrong, but this is causing me some trouble.  Do I need to link something in order for this to work?

<img width="1003" alt="Screen Shot 2020-10-05 at 11 27 53 AM" src="https://user-images.githubusercontent.com/10561050/95102041-90477880-073b-11eb-9d89-98961706d21f.png">

This was after running `npm start` from the newly created extension directory.

Reply:
Thanks for taking this for a spin @joshuatf. The problem lies step 3 of the instructions, which assumed the sibling wc-admin plugin had built packages. I changed the instructions to reflect this if you could give it another shot.

The trouble lies in creating a dependency on a package that isn't published yet, so this will be a lot simpler once the package in on npm.